Dockerfile.base

```diff
@@ -13,6 +13,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 ENV DEBIAN_FRONTEND=noninteractive
 # hadolint ignore=DL3003,DL4001,SC2086
 RUN echo en_US.UTF-8 UTF-8 >> /etc/locale.gen \
+  && sed -e 's/Defaults	env_reset/Defaults	env_keep = "HTTP_PROXY HTTPS_PROXY NO_PROXY FTP_PROXY http_proxy https_proxy no_proxy ftp_proxy"/' -i /etc/sudoers
```

Comment:
This needs to end with a `\`

Reply:
Totally my fault. This is what I get for copy and pasting from my local extended image. Sorry!